### PR TITLE
met MVP after much turmoil

### PR DESCRIPTION
**Pull request by micahjones13**
I met MVP : )

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,16 +1,72 @@
...
+  }
+  
   render() {
     return (
       <div>
         <h2>Welcome to your Todo App!</h2>
+        <div>
+        {
+            
+            this.state.todoData.map(itemFromMap => (
```
Comment (Joshua-Wood39) [inline src/App.js:52]: Excellent use of map to iterate out your components!

File: src/components/TodoComponents/TodoData.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+//toDo array
+
+export const todoArr = [
```
Comment (Joshua-Wood39) [inline src/components/TodoComponents/TodoData.js:3]: I like that you turned your data into a component. That makes your code easier to modify and understand.

File: src/components/TodoComponents/TodoForm.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+
+
+
+// class TodoForm extends React.Component {
```
Comment (Joshua-Wood39) [inline src/components/TodoComponents/TodoForm.js:6]: Did you run into issues having more than one Class component??

**Review by Joshua-Wood39 — APPROVED** [review]
An excellent foundation for today's material. Good work!

File: src/components/TodoComponents/TodoForm.js (hunk 3 above)
Comment (micahjones13) [reply src/components/TodoComponents/TodoForm.js:6]: I did for day 2 of the project, and ended up changing it to having only one class component.

**Pull request merged**